.github/scripts/validate-bicep.sh

```diff
@@ -4,12 +4,13 @@ then
     BICEP_EXECUTABLE="$BICEP_PATH/rad-bicep"
 fi
 
+WARNING_MSG="WARNING: The following experimental Bicep features"
```

Comment:
Could you please add some context on what it is/how it is being consumed?

Reply:
Updated